<?xml version="1.0" encoding="utf-8"?>
<launch>

    ### ROSBRIDGE ###
    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>

    ### Semantic_maping Virtual###
    <node name="semantic_mapping" pkg="vimantic" type="ViMantic_VirtualHome_Detectron_node.py" output="screen">
        <param name="topic_result" value="ViMantic/SemanticObjects"/>
        <param name="topic_virtualCameraRGBD" value="ViMantic/VirtualCameraRGBD"/>
        <param name="topic_republic" value="ViMantic/ToCNN"/>
        <param name="topic_cnn" value="detectron2_ros/result"/>
        <param name="debug" value="true"/>
        <param name="record_times" value="false"/>
    </node>

    ### RVIZ ###
    <node name="rviz_player" pkg="rviz" type="rviz"
          args="-d $(find vimantic)/launch/ViMantic_virtualHome_detectron.rviz"/>


</launch>
